manifests/extension.pp

```diff
@@ -180,8 +184,9 @@
 
   $config_root_ini = pick_default($::php::config_root_ini, $::php::params::config_root_ini)
   ::php::config { $title:
-    file   => "${config_root_ini}/${lowercase_title}.ini",
-    config => $final_settings,
+    file    => "${config_root_ini}/${lowercase_title}.ini",
+    config  => $final_settings,
+    require => Package[$real_package],
```

Comment:
I know it's working also here, but now having the dependency at all when this is empty is IMHO better.